frontstage/views/passwords.py

```diff
@@ -92,32 +70,36 @@ def forgot_password_check_email():
 @passwords_bp.route('/reset-password/<token>', methods=['GET', 'POST'])
```

Comment:
We should split the GET and POST handlers out into two methods really. Appreciate it wasn't introduced in your changes